tests/unittests/test_net.py

```diff
@@ -7693,14 +7697,27 @@ class TestGetInterfacesByMac(CiTestCase):
             "bridge1-nic": "aa:aa:aa:aa:aa:03",
             "lo": "00:00:00:00:00:00",
             "greptap0": "00:00:00:00:00:00",
+            "netvsc0-vf": "aa:aa:aa:aa:aa:04",
+            "netvsc0": "aa:aa:aa:aa:aa:04",
+            "netvsc1-vf": "aa:aa:aa:aa:aa:05",
+            "netvsc1": "aa:aa:aa:aa:aa:05",
             "tun0": None,
         },
+        "drivers": {
+            "netvsc0": "hv_netvsc",
+            "netvsc0-vf": "foo",
+            "netvsc1": "hv_netvsc",
+            "netvsc1-vf": "bar",
```

Comment:
Is this verifying that the `-vf` interfaces doesn't cause existing calls to `get_interfaces_by_mac()` to throw an exception?

Reply:
Initially I assumed you forgot to push another test commit.

Reply:
@holmanb  Yep, that is correct. The existing tests will provide coverage with the added test data.